common/components/ui/UnlockHeader.tsx

```diff
@@ -57,7 +60,7 @@ export class UnlockHeader extends React.Component<Props, State> {
     );
   }
 
-  public toggleisExpanded = () => {
+  public toggleisExpanded = (_: React.FormEvent<HTMLButtonElement>) => {
```

Comment:
If it's unused, is there any reason in specifying?

Reply:
@HenryNguyen5 ^

Reply:
It's just to specify what it's being used on as an event handler.